account_invoice_date/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+#. Go to Invoice > Click Create > Create new invoice.
+#. Click on "Confirm" button.
+#. Select the date and Click on "Open invoice" button.
```

Comment:
```suggestion
#. Select the date and Click on "Confirm" button.
```

Minor thing before the merge

Reply:
Also it should be "click".

Reply:
@AaronHForgeFlow, Changes are done, Kindly review it.